src/sidebar/services/api.js

```diff
@@ -132,7 +160,11 @@ function createAPICall($http, $q, links, route, tokenGetter) {
       };
       return $http(req);
     }).then(function (response) {
-      return response.data;
+      if (options.includeMetadata) {
+        return { data: response.data, token };
+      } else {
+        return response.data;
```

Comment:
Hrm, I am quite likely missing a step here, but does this return value conform to the `APIResponse` `typedef`?

Reply:
The type of the response is `Promise<APIResponse|any>`. This is the `any` path.